Example/SmileID/HomeViewModel.swift

```diff
@@ -86,6 +86,13 @@ class HomeViewModel: ObservableObject, SmartSelfieResultDelegate, DocumentCaptur
                                                object: nil)
     }
 
+    func didSucceed(selfie: Data,
+                     documentFrontImage: Data,
```

Comment:
nit: spacing. Is it possible for the linter to catch this?
```suggestion
                    documentFrontImage: Data,
```